server/utils/placements/index.ts

```diff
@@ -77,6 +64,12 @@ export const getBackLink = (referrer: string, premisesId: string): string => {
   return paths.premises.show({ premisesId })
 }
 
+const summaryRow = (key: string, value: string, excludeRow = false): SummaryListItem =>
+  value && {
+    key: excludeRow ? null : textValue(key),
+    value: textValue(value),
+  }
+
```

Comment:
If this now checks the value is truthy, there should be no need for `excludeRow`, and the uses below shouldn't need the third argument either